app/models/job_template.rb

```diff
@@ -87,6 +88,20 @@ def effective_user
     super || (build_effective_user.tap(&:set_defaults))
   end
 
+  def generate_description_format
+    if description_format.blank?
+      names = template_inputs.map(&:name)
+      if names.empty?
+        '%{job_name}'
+      else
+        inputs = names.map { |name| %Q(#{name}="%{#{name}}") }.join(' ')
+        "%{job_name} with inputs #{inputs}"
```

Comment:
`input_names` name would be more descriptive than just `names`. Also, to reduce the duplicity, it could look like this:

```ruby
generated_description = '%{job_name}'
unless template_inputs.empty?
  inputs = template_inputs.map(&:name}.map { |name| %Q(#{name}="%{#{name}}") }.join(' ')
  generated_description << " with inputs #{inputs}"
end
generated_description
```